web-console/src/views/load-data-view/load-data-view.tsx

```diff
@@ -1174,12 +1174,14 @@ export class LoadDataView extends React.PureComponent<LoadDataViewProps, LoadDat
                 );
               }
 
+              newSpec = deepSet(newSpec, 'spec.ioConfig.disableNullColumnSkipping', true);
+
               this.updateSpec(fillDataSourceNameIfNeeded(newSpec));
             } else {
               this.updateSpec(
                 fillDataSourceNameIfNeeded(
                   fillInputFormat(
-                    spec,
+                    deepSet(spec, 'spec.ioConfig.disableNullColumnSkipping', true),
```

Comment:
Don't actually set this in the `spec` that will be sent to Druid. Only on the specs sent to the sampler